barretenberg/cpp/src/barretenberg/dsl/acir_format/serde/witness_map.txt

```diff
@@ -0,0 +1,108 @@
+#pragma once
```

Comment:
noting that this is a new file

Reply:
Likely we want to rename the extension to .hpp

Reply:
Does this file needs to be generated?
It was in the folder with acir.cpp, so I thought it was generated but then I removed it (and kept the motif in this txt file) and re-generated the acipr.cpp and it was not modified, so I did not touch it in BB/dsl (except for this txt file that I am removing now.)